src/helm/proxy/clients/together_client.py

```diff
@@ -182,8 +186,7 @@ def make_request(self, request: Request) -> RequestResult:
             raise TogetherClientError("togetherApiKey not set in credentials.conf")
         headers: Dict[str, str] = {"Authorization": f"Bearer {self.api_key}"}
 
-        # TODO: Remove synchronous branch.
-        if request.model_engine in MODEL_ALIASES:
+        if TOGETHER_SUPPORTS_ASYNC_REQUESTS:
```

Comment:
Shouldn't you add `and request.model_engine in MDOEL_ALIASES` ?

Reply:
Probably not. Together is probably going to enable async for everything or for nothing.